Dockerfile

```
@@ -0,0 +1,25 @@
+FROM node:24-bookworm-slim AS builder
```

Bun offer an official image, rather than installing as soon as you grab the node image
```suggestion
FROM oven/bun:1-debian AS builder
```